js/decklist/dci.js

```diff
@@ -0,0 +1,71 @@
+var primes = [43,47,53,71,73,31,37,41,59,61,67,29];
+function isValid(number){
```

Comment:
Maybe these functions should be more descriptively named? Or perhaps nested so that they don't bleed into global namespace?